Fabulous.XamarinForms/samples/FabulousWeather/FabulousWeather/App.fs

```diff
@@ -0,0 +1,154 @@
+﻿namespace FabulousWeather
+
+open Fabulous
+open Fabulous.XamarinForms
+open Fabulous.XamarinForms.LiveUpdate
+open Xamarin.Forms
+
+module App =
+    type Operator = Add | Subtract | Multiply | Divide 
+
+    /// Represents a calculator button press
+    type Msg =
+        | Refresh
+
+    
+    type DataItem = {Name:string;Value:int}
+    type Model = {Temp:int;Items: DataItem list}
+
+    let initial = {Temp=61;Items=[
+        {Name="Pressure";Value=10}
+        {Name="UV Index";Value=3}
+        {Name="Wind Speed";Value=0}
+        {Name="Humidity";Value=65}
+        {Name="Min Temp";Value=50}
+        {Name="Max Temp";Value=80}
+        ]}
+     
+    let update msg model =
+        match msg with
+        | Refresh -> initial
+
+    let view (model: Model) dispatch =
+        let coldStartColor = Color.FromHex("#BDE3FA")
+        let coldEndColor = Color.FromHex("#A5C9FD")
+        let WarmStartColor =  Color.FromHex("#F6CC66")
+        let WarmEndColor    = Color.FromHex("#FCA184")
+        let NightStartColor = Color.FromHex("#172941")
+        let NightEndColor =   Color.FromHex("#3C6683")
+        let MainTextColor =   Color.White
+        let itemStartColor = Color.FromHex("#98FFFFFF")
+        let itemEndColor = Color.FromHex("#60FFFFFF")
+
+
+        
+
+        let createLabel value =
+            View.Label(text=value).FontSize(FontSize 100.).HorizontalOptions(LayoutOptions.Center).TextColor(MainTextColor)
+
+        let getColor isStart temp =
+            if temp > 60 then 
+                if isStart then WarmStartColor else WarmEndColor
+            else if temp = -100 then 
+                if isStart then NightStartColor else NightEndColor
+            else if isStart then 
+                coldStartColor else coldEndColor
+        
+        let itemsView =
+            [for r in model.Items -> 
+                  View.PancakeView(content=
+                        View.Label(text=r.Name + "\r\n" + r.Value.ToString(),textColor=MainTextColor),
+                        cornerRadius=new CornerRadius(20.,20.,20.,0.),
```

Comment:
The `new` keywords can be safely removed here.
F# only complains about `new` when it's missing on disposable values.